sympy/solvers/inequalities.py

```diff
@@ -1004,3 +1003,233 @@ def reduce_inequalities(inequalities, symbols=[]):
+        inequalities = _merge_lower_greater(greater_than, lower_than) + extra
+        pivot = _find_pivot(inequalities)
+    return inequalities, res
+
+
+def _fourier_motzkin_extension(inequalities):
+    """Extension of the Fourier-Motzkin algorithm to the case where inequalities
+    do not contain variables that have at least two coefficients with opposite sign.
+
+    Examples
+    ========
+
+    >>> from sympy.solvers.inequalities import _fourier_motzkin_extension
+    >>> from sympy.abc import x, y, z
+
+    >>> eq1 = 2*x - 3*y + z + 1
+    >>> eq2 = x - y + 2*z - 2
+    >>> eq3 = x - y + 3*z + 4
+    >>> eq4 = x + z
+
+    >>> inequalities = [eq1, eq2, eq3, eq4]
+    >>> d = _fourier_motzkin_extension(inequalities)
+    >>> assert set(d) == {x}
+    >>> d[x]
+    {'greater_than': [3*y/2 - z/2 - 1/2, y - 2*z + 2, y - 3*z - 4, -z], 'lower_than': []}
+    """
+
+    pivot = _pick_var(inequalities)
+    res = {}
+    while inequalities:
+        greater_than, lower_than, extra = _split_lower_greater(inequalities, pivot)
+        res[pivot] = {"greater_than":greater_than, "lower_than":lower_than}
+        inequalities = extra
+        pivot = _pick_var(inequalities)
+    return res
+
+
+def _pick_var(inequalities):
+    """Return a free variable of the system of inequalities
+
+    Examples
+    ========
+
+    >>> from sympy.solvers.inequalities import _pick_var
+    >>> from sympy.abc import x, y, z
+
+    >>> eq1 = 2*x - 3*y + z + 1
+    >>> eq2 = x - y + 2*z - 2
+    >>> eq3 = x + y + 3*z + 4
+    >>> eq4 = x - z
+
+    >>> inequalities = [eq1, eq2, eq3, eq4]
+    >>> _pick_var(inequalities)
+    x
+
+    """
+    for eq in inequalities:  # should already be in canonical order
+        for symb in ordered(eq.free_symbols):  # make selection canonical
+            return symb
+
+
+def solve_linear_inequalities(inequalities):
+    """ Solve a system of linear inequalities
+
+    Parameters
+    ==========
+
+    inequalities: list of sympy equations
+        The system of inequalities to solve. All equations in the list are assume to be
+        linear and greater than 0. The system must be expressed as follows:
+
+        2x - 3y +  z + 1 > 0
+        x  -  y + 2z - 2 > 0
+        x  +  y + 3z + 4 > 0
+        x       -  z     > 0
+
+    Examples
+    ========
+
+    >>> from sympy.solvers.inequalities import solve_linear_inequalities
+    >>> from sympy.abc import x, y, z
+
+    >>> eq1 = 2*x - 3*y + z + 1
+    >>> eq2 = x - y + 2*z - 2
+    >>> eq3 = x + y + 3*z + 4
+    >>> eq4 = x - z
+
+    >>> inequalities=[eq1, eq2, eq3, eq4]
+    >>> d = solve_linear_inequalities(inequalities)
+    >>> assert set(d) == set([x, y, z])
+    >>> d[x]
+    {'greater_than': [-13/15, -2/7], 'lower_than': []}
+    >>> d[y]
+    {'greater_than': [-x - 3*z - 4], 'lower_than': [2*x/3 + z/3 + 1/3, x + 2*z - 2]}
+    >>> d[z]
+    {'greater_than': [-x/2 - 13/10, -2*x/5 - 2/5], 'lower_than': [x]}
```

Comment:
Failed example:
    d[x]
Expected:
    {'greater_than': [-13/15, -2/7], 'lower_than': []}
Got:
    {'greater_than': [-2/7, -13/15], 'lower_than': []}
**********************************************************************
File "/home/runner/work/sympy/sympy/sympy/solvers/inequalities.py", line 1223, in sympy.solvers.inequalities.solve_linear_inequalities
Failed example:
    d[y]
Expected:
    {'greater_than': [-x - 3*z - 4], 'lower_than': [2*x/3 + z/3 + 1/3, x + 2*z - 2]}
Got:
    {'greater_than': [-4*x - 4], 'lower_than': [3*x - 2, x + 1/3]}
**********************************************************************
File "/home/runner/work/sympy/sympy/sympy/solvers/inequalities.py", line 1225, in sympy.solvers.inequalities.solve_linear_inequalities
Failed example:
    d[z]
Expected:
    {'greater_than': [-x/2 - 13/10, -2*x/5 - 2/5], 'lower_than': [x]}
Got:
    {'greater_than': [-x/3 - y/3 - 4/3, -x/2 + y/2 + 1, -2*x + 3*y - 1], 'lower_than': [x]}


```suggestion
    >>> d[x]
     {'greater_than': [-2/7, -13/15], 'lower_than': []}
    >>> d[y]
    {'greater_than': [-4*x - 4], 'lower_than': [3*x - 2, x + 1/3]}
    >>> d[z]
     {'greater_than': [-x/3 - y/3 - 4/3, -x/2 + y/2 + 1, -2*x + 3*y - 1], 'lower_than': [x]}
```